requirements/software-requirements.md

```diff
@@ -233,6 +257,7 @@ local Mac or Windows machine.
    IR6).
 5. Application data (state) that needs to be persistent between runs *MUST* be stored separate from the container (e.g. in a
    mounted volume) (See point 3 of the [Infrastructure Requirements (IR)](infrastructure-requirements.md))
+6. The container build process *SHOULD* make use proper packaging ecosystems for installing its individual components. Providing a container *SHOULD NOT* be considered a substitute for component packaging (see point 6, which is to be considered as complementary to or a prerequisite for this one).
```

Comment:
```suggestion
6. The container build process *SHOULD* use proper packaging ecosystems for installing its individual components. Providing a container *SHOULD NOT* be considered a substitute for component packaging (see point 6, which is to be considered as complementary to or a prerequisite for this one).
```

Reply:
I don’t completely agree with this. To me, the distinction applications (published as container images) and libraries (published as language-specific packages) makes more sense.